README.md

```diff
@@ -254,6 +254,53 @@ curl http://localhost:23333/v1/chat/completions \
 
 Find more details in the [LMDeploy documentation](https://lmdeploy.readthedocs.io/en/latest/)
 
+#### SGLang inference
+
+##### Installation
+```bash
+pip3 install "sglang[srt]>=0.4.1.post6" --find-links https://flashinfer.ai/whl/cu124/torch2.4/flashinfer/
+```
+
+##### OpenAI Compatible Server
+
+```bash
+python3 -m sglang.launch_server --model internlm/internlm3-8b-instruct --trust-remote-code --chat-template internlm2-chat
+```
+
+##### cURL client
```

Comment:
Let's keep one client example in README to prevent excessive information.
We can leave the sglang's guide link for user to access  more information

Reply:
done